src/app.rs

```diff
@@ -120,8 +120,8 @@ pub fn build() -> App<'static, 'static> {
             Arg::with_name("directory-only")
                 .short("d")
                 .long("directory-only")
-                .conflicts_with("all")
-                .conflicts_with("almost-all")
+                //.conflicts_with("all")
```

Comment:
There is no reason to keep the code commented. We can complete drop it.

Reply:
You right, gonna remove it.